pliers/extractors/text.py

```diff
@@ -313,3 +335,38 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+class SpaCyExtractor(TextExtractor):
+    
+    def __init__(self, model='en_core_web_sm', features=None):
```

Comment:
Don't forget to store the passed arguments internally in the instance (i.e., to assign `self.model = model`, etc.) That way you can access them later from the `_transform` method.